app/app.go

```diff
@@ -87,6 +84,9 @@ import (
 	ibcporttypes "github.com/cosmos/ibc-go/v3/modules/core/05-port/types"
 	ibchost "github.com/cosmos/ibc-go/v3/modules/core/24-host"
 	ibckeeper "github.com/cosmos/ibc-go/v3/modules/core/keeper"
+	"github.com/sei-protocol/sei-chain/x/mint"
```

Comment:
basically we want to make sure all imports are migrated from
"github.com/cosmos/cosmos-sdk/x/mint" -> "github.com/sei-protocol/sei-chain/x/mint"
A lot of these within the x/mint module, but mostly just imports